component/client-handler/src/main/java/org/wso2/carbon/identity/oauth2/token/handler/clientauth/jwt/validator/JWTValidator.java

```diff
@@ -399,8 +399,9 @@ private boolean isValidSignature(String clientId, SignedJWT signedJWT, String te
                     isValidSignature = new JWKSBasedJWTValidator().validateSignature(jwtString, jwksUri, alg, options);
                 }
             } catch (IdentityOAuth2Exception e) {
-                String errorMessage = "Error occurred while validating signature using jwks";
-                throw new OAuthClientAuthnException(errorMessage, OAuth2ErrorCodes.INVALID_REQUEST, e);
+                String errorMessage = "Error occurred while validating signature using jwks ";
```

Comment:
Remove the space after "jwks"

Reply:
fixed in https://github.com/wso2-extensions/identity-oauth-addons/commit/8ba88067e171f3bbaa355f2f2f503585d4fb428c